app/src/twilio/java/com/twilio/video/app/VideoApplicationComponent.java

```diff
@@ -30,7 +30,8 @@
         SettingsActivityModule.class,
         SettingsFragmentModule.class,
         VideoServiceModule.class,
-        RoomManagerModule.class
+        RoomManagerModule.class,
+        AudioRouterModule.class
```

Comment:
You may consider naming this AudioSwitch module

Reply:
Ah yep. Thanks!